src/class-fields/propname-error/cls-expr-computed-name.template

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-class-definitions-static-semantics-early-errors
+path: language/expressions/class/fields-computed-name-
+name: computed
+info: |
+  Static Semantics: PropName
+  ...
+  ComputedPropertyName : [ AssignmentExpression ]
+    Return empty.
+---*/
+
+x = /*{ propname }*/;
```

Comment:
var, let or const?

Reply:
this line should be like `var x = "/*{ propname }*/";